### ISSUE TITLE
Support for account-management services by policy resources

### ISSUE THREAD

**Issue by ecneladis** [IBM-Cloud/terraform-provider-ibm#500]
It looks like [ibm_iam_group_policy resource](https://ibm-cloud.github.io/tf-ibm-docs/v0.14.1/r/iam_access_group_policy.html) doesn't support `account-management` services required to manage IAM. Same for user and service policies.

I tried to find it with `ibmcloud catalog search` as described in doc, but without success:
![image](https://user-images.githubusercontent.com/6756744/51908979-a67c2d80-23cb-11e9-897b-aa993bfe60e0.png)

In `ibmcloud` cli there's a separate option:
```
    ibmcloud iam access-group-policy-create example_group --roles Administrator --account-management
        Give example_group Administrator role for all account management services
```

**Comment by hkantare** [comment]
Fix is available in latest releases v0.16.0 and v0.16.1.

**Comment by hkantare** [comment]
closing the issue
